src/app/verification-addresses/features/user-iva-list/user-iva-list.component.html

```diff
@@ -31,7 +31,7 @@
           <div class="text-center">
             @switch (iva.state) {
               @case ('Verified') {
-                <div class="text-bold text-base text-green-500">
+                <div class="text-bold text-success text-base">
```

Comment:
es-lint wants to change the order here

Reply:
I know but prettier doesn't do it.

Reply:
Strange, it works for me. 

In the settings I have
```
  "editor.defaultFormatter": "esbenp.prettier-vscode",
  "editor.formatOnSave": true,
```

Should also be fixed with `npm run lf`.

Reply:
`Replace `base·text-success` with `success·text-base`eslint[prettier/prettier](https://github.com/prettier/eslint-plugin-prettier#options)`

Is what I get from Prettier.

Reply:
Yes, I see this in the IDE or when I run "npm run lint". But when I save, of use the "Quick fix..." or run "npm run lf" then the order is reversed and prettier is happy.